news/_posts/performance/2020-12-07-performance-and-prestashop.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "Performance (& PrestaShop)"
+subtitle: "Sorry, Goose, but it's time to buzz the tower."
+date:   2020-12-04 9:00:00
+authors: [ Claude-ArnaudPerrot ]
+icon: icon-airplane
+tags: [performance, whitepaper]
+---
+
+Introducing a new series of article to talk about performance, benchmarking and PrestaShop!
+
+
+# Performance (& PrestaShop)
+
+You may have heard of it, but we'll soon release a White Paper, which is all about PrestaShop's performance, how to benchmark it and how it behaves under loads. And what to do to keep it responding under 300ms.
+
+Still, there are a number of matters that can't be covered in a White Paper - hopefully the first in a series, fingers crossed.
+Or that can't be given the space we think they deserve.
```

Comment:
Markdown jumps a line if there is a fully blank line. Did you want to make a line break? It displays like this https://github.com/PrestaShop/prestashop.github.io/blob/eba12a430272eb860b2a78f6d8fcb811f5d02d73/news/_posts/performance/2020-12-07-performance-and-prestashop.md

Reply:
If you want line break you need to do
```
Still, there are a number of matters that can't be covered in a White Paper - hopefully the first in a series, fingers crossed.

Or that can't be given the space we think they deserve.
```

Reply:
Good catch, because I did think about jumping lines, but decided against it in the end, as it would give a far too light text to my taste ;)